IDEAS/Buildings/Components/Window.mo

```diff
@@ -72,18 +76,29 @@ model Window "Multipane window"
 
 
 
-  parameter Real coeffsCp[:,:]=[0,0.4; 45,0.1; 90,-0.3; 135,-0.35; 180,-0.2; 225,
-      -0.35; 270,-0.3; 315,0.1; 360,0.4]
-      "Cp at different angles of attack"
-      annotation(Dialog(tab="Airflow",group="Wind"));
+  replaceable parameter
+    IDEAS.Buildings.Data.WindPressureCoeff.Lowrise_Square_Exposed Cp_table
+    constrainedby IDEAS.Buildings.Data.Interfaces.WindPressureCoeff
+    "Tables with wind pressure coefficients for walls, floors and roofs"
+    annotation (
+    __Dymola_choicesAllMatching=true,
+    Placement(transformation(extent={{-34,78},{-30,82}})),
+    Dialog(tab="Airflow", group="Wind Pressure"));
+
+  parameter Real coeffsCp[:,:]= if inc<=Modelica.Constants.pi/18 then Cp_table.Cp_Roof_0_10 elseif inc<=Modelica.Constants.pi/6  then  Cp_table.Cp_Roof_11_30 elseif inc<=Modelica.Constants.pi/4 then Cp_table.Cp_Roof_30_45 elseif  inc==Modelica.Constants.pi then Cp_table.Cp_Floor else Cp_table.Cp_Wall
+      "Cp at different angles of attack, default the correct table will be selected from Cp_table based on the surface tilt"
+      annotation(Dialog(tab="Airflow", group="Wind Pressure"));
+
+  parameter Boolean Use_custom_Cs = false
```

Comment:
Small u

Reply:
I think there is code duplication here. We may want to put this in the partial and have `final use_custom_Cs=false` in models where the parameter has no meaning.